source/material_model/latent_heat.cc

```diff
@@ -228,6 +229,7 @@ namespace aspect
                                                                  i);
 
                     phase_dependence += phaseFunction * density_jumps[i];
+                    viscosity_phase_dependence *= 1 + phaseFunction * (phase_prefactors[i+1]-1);
```

Comment:
why do you use `phase_prefactors[i+1]` but `density_jumps[i]`? Maybe there is a reason, but this asymmetry looks strange.